<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apps</class>
 <widget class="QWidget" name="apps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="5" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>407</width>
          <height>566</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Built-in apps</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="Line" name="line">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>3</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="verticalSpacing">
              <number>10</number>
             </property>
             <item row="3" column="1">
              <widget class="QPushButton" name="calendarLaunchBtn">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="g2048Label">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>2048</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QPushButton" name="todoLaunchBtn">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="calculatorLaunchBtn">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="reversiLaunchBtn">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="savedWordsLaunchBtn">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="vncLaunchBtn">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="calendarLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Calendar</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="g2048LaunchBtn">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="reversiLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Reversi</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Qalculate!</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Saved words</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="todoLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>To-Do</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>KoBox apps</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="vncViewerLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>VNC viewer</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="koboxAppsOpenButton">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Open</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QPushButton" name="flashExamLaunchBtn">
               <property name="font">
                <font>
                 <family>Inter</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Launch</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="flashExamLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>FlashExam</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>User apps</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="editUserAppsBtn">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="topMargin">
            <number>12</number>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </item>
         <item>
          <widget class="QLabel" name="noUserAppsAvailableLabel">
           <property name="font">
            <font>
             <family>Chivo</family>
            </font>
           </property>
           <property name="text">
            <string>
No user apps currently available</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
